usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -283,7 +283,7 @@ function make_syslinux_config {
         syslinux_menu "TABMSG Press [Tab] to edit options and [F1] for version info"
     fi
 
-    echo "timeout 300"
+    echo "timeout $ISO_SYSLINUX_TIMEOUT"
```

Comment:
Instead of introducing a new variable I would place the calculation here:

```suggestion
    echo "timeout $(( $USER_INPUT_TIMEOUT * 10 ))"
```

This way it will always do what the user meant.

Reply:
I would keep this separate. In my case I set the `ISO_SYSLINUX_TIMEOUT` to 0 and `USER_INPUT_TIMEOUT` to 10

Reply:
I see. Then maybe set `ISO_SYSLINUX_TIMEOUT=""` in `default.conf`and use `USER_INPUT_TIMEOUT` as default value for that? Maybe like this:

```shell
echo "timeout ${ISO_SYSLINUX_TIMEOUT:-$(( $USER_INPUT_TIMEOUT * 10 ))}"
```

What do you think? Dynamic defaults and the ability to override if needed.

Reply:
Yes, this way it is right because
when config variables are empty
then ReaR can do things on its own.